.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Basilisk Official Documentation
+    url: http://basilisk.fr/
+    about: Check the official Basilisk documentation for detailed information
+  - name: CoMPhy Lab
+    url: https://comphy-lab.org/
+    about: Visit the Computational Multiphase Physics Lab website 
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)